src/components/Garden/Feed/ProposalCardModalSupporters.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo } from 'react'
+import { GU, Modal, textStyle, useViewport } from '@1hive/1hive-ui'
+import { ProposalType } from '@/hooks/constants'
+import { useGardenState } from '@/providers/GardenState'
+import IdentityBadge from '@/components/IdentityBadge'
+import { formatTokenAmount } from '@utils/token-utils'
+
+type ModalSupportersProps = {
+  proposal: ProposalType
+  visible: boolean
+  onClose: () => void
+}
+
+const ModalSupporters = ({
+  proposal,
+  visible,
+  onClose,
+}: ModalSupportersProps) => {
+  const { width } = useViewport()
+  const { config } = useGardenState()
+  const { requestToken } = config.conviction
+
+  // Show stakes with more than 0.01HNY
+  const filteredStakesByAmount = useMemo(() => {
+    return proposal?.stakes.filter(
+      (stake) =>
+        formatTokenAmount(stake?.amount, requestToken.decimals) > '0.01'
```

Comment:
```suggestion
        formatTokenAmount(stake?.amount, requestToken.decimals) >= '0.01'
```

Reply:
Also, what do you think of including a sort of `stake.amount` in descending order?
```
sort(
    (stake1, stake2) =>
      parseInt(formatTokenAmount(stake2.amount, requestToken.decimals)) -
      parseInt(formatTokenAmount(stake1.amount, requestToken.decimals))
)
```

Reply:
Yes I could do that as well 👌